mdnDocsOverrides.json

```
@@ -796,8 +796,12 @@
   "fetch": [
     "Web/API/Window/fetch"
   ],
-  "fetch-metadata": [],
-  "fetch-priority": [],
+  "fetch-metadata": [
+    "Glossary/Fetch_metadata_request_header"
+  ],
+  "fetch-priority": [
+    "Web/API/HTMLScriptElement/fetchPriority"
+  ],
```

For HTML elements, and HTML element attributes, I think it's better to link to the Web/HTML/Element path instead of the corresponding Web/API/HTML___Element path. The former is about HTML elements, the latter is about the APIs exposed by these elements. I think the former is easier to understand.
We can probably link to Fetch too, since the method supports the option as well (although there's no section dedicated to it in the docs).

```suggestion
  "fetch-priority": [
    "Web/HTML/Element/script#fetchpriority",
    "Web/HTML/Element/link#fetchpriority",
    "Web/HTML/Element/img#fetchpriority",
    "Web/API/Window/fetch"
  ],
```

Note: for now, on the explorer, the link texts that get generated from these URL slugs match the overall page title. This isn't great in cases where we target a specific anchor on the page. I'll fix this by doing something like `See <anchor> on <title of page>`.